operations/multi-tenancy/Create_a_Tenant.md

```diff
@@ -281,7 +281,9 @@ spec:
           memory: 512Mi
 ```
 
-**Note:** Container versions must be customized to the versions installed on the system
+**Note:** 
+- Container versions must be customized to the versions installed on the system
+- Please set spec.munge.uid to 498 and spec.munge.gid to 484 if using munger container version cray/munge-munge:1.6.0
```

Comment:
Remove trailing space on `**Note:**` and I suggest using a block comment to callout the note.
```suggestion
> **Note:**
> - Container versions must be customized to the versions installed on the system
> - Please set spec.munge.uid to 498 and spec.munge.gid to 484 if using munger container version cray/munge-munge:1.6.0
```